src/status_im/chat/events/commands.cljs

```diff
@@ -41,7 +41,8 @@
             to          (get-in contacts [chat-id :address])
             jail-params {:parameters params
                          :context    (generate-context db chat-id to group-id)}]
-        {:call-jail {:jail-id                 jail-id
+        {:db db
```

Comment:
somewhere a little perfectionist cries :)

Reply:
It depends on the kind of perfectionnist, I consider myself one but I cry everytime I see all these extra spaces everywhere :laughing: